corehq/apps/app_manager/models.py

```diff
@@ -987,6 +987,7 @@ def wrapped_xform(self):
     def validate_form(self):
         vc = self.validation_cache
         if vc is None:
+            # todo: now that we don't use formtranslate, does this still apply?
```

Comment:
OK, sounds like it's definitely fine to leave the stripping, and possibly not fine to remove it. I'm  tempted to leave both the original comment and the todo comment, so if at any point someone traces an issue back to this line they can see both why it was put there in the first place and a comment suggesting that it's possibly not needed anymore; then they can figure out if removing it works fine/fixes their issue.